debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -484,7 +484,7 @@ public Map<TableId, String> getSnapshotSelectOverridesByTable() {
     private static int validateDatabaseNames(Configuration config, Field field, Field.ValidationOutput problems) {
         String databaseNames = config.getString(field);
         int count = 0;
-        if (databaseNames == null || databaseNames.split(",").length == 0) {
+        if (databaseNames == null || databaseNames.isEmpty() || databaseNames.split(",").length == 0) {
```

Comment:
Doesn't the new condition cover the last one? It looks like it does, according to the tests. The whole condition could be simplified further to:
```suggestion
        if (Strings.isNullOrBlank(databaseNames)) {
```

Reply:
> As I understand, validateDatabaseNames() is invoked upon connection configuration

As we confirmed offline this is invoked inside `BaseSourceTask#start` which occurs prior to the `SqlServerConnectorTask#start` and this should already be validated. As discussed, I've added another commit to stop reparsing the `DATABASE_NAMES`.